web/src/routes/Storage.jsx

```diff
@@ -18,95 +18,118 @@ export default function Storage() {
 
   const { service } = stats || initialStats || emptyObject;
 
+  if (!service || !storage) {
+    return <ActivityIndicator />;
+  }
+
+  let storage_usage;
+  if (
+    service &&
+    service['storage']['/media/frigate/recordings']['total'] != service['storage']['/media/frigate/recordings']['total']
```

Comment:
This if statement appears to be checking if it is not equal to itself. What's the advantage of not displaying both all of the time?

Reply:
so does this only give the value of the used/total for the drive itself, not the individual folders?

Reply:
Yes, otherwise it would require spinning up the drive every time stats were checked which is not okay.

Reply:
That makes sense. Thanks for the clarification!